### DISCUSSION TITLE
Interaction to Next Paint (INP)

### DISCUSSION THREAD

**Discussion by andriiradkevych** [freshframework/fresh#2347] [General]
Google added new `Core Web Vitals` called `INP` , and it seems will be a problem with `fresh` . Here you can find description https://web.dev/articles/inp . It is measuring interaction , and clicking on urls , is also as I understand interaction , which is slow on `fresh` . Is there any ways to improve it ?

<img width="259" alt="Screenshot 2024-03-03 at 16 19 40" src="https://github.com/denoland/fresh/assets/26725494/8611a5d2-0697-4eeb-8e1b-3e82c0d3e317">

**Comment by marvinhagemeister** [comment]
Are you sure that the problem is Fresh itself? Fresh is quite speedy and in most cases when it comes to performance the problem is not the framework itself.
